contracts/treasury/src/execute.rs

```diff
@@ -85,3 +90,59 @@ pub fn execute_accept_ownership(
         None => Err(ContractError::NoPendingOwner {}),
     }
 }
+
+pub fn execute_spend_funds(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    amount: cosmwasm_std::Coin,
+    receiver: String,
+    channel_id: Option<String>,
+) -> ContractResult<Response> {
+    ADMIN.assert_admin(deps.as_ref(), &info.sender)?;
+
+    let msg_send: CosmosMsg;
+
+    if channel_id.is_none() {
+        msg_send = cosmwasm_std::BankMsg::Send {
+            to_address: receiver.clone(),
+            amount: vec![amount.clone()],
+        }
+        .into();
+    } else {
+        // not using the ibc queue here. if this fails, we just reexecute
+        msg_send = MsgTransfer {
+            source_channel: channel_id.clone().unwrap().clone(),
+            source_port: "transfer".to_string(),
+            token: Some(Coin {
+                denom: amount.clone().denom,
+                amount: amount.clone().amount.to_string(),
+            }),
+            receiver: receiver.clone(),
```

Comment:
It appears that the receiver's address must be a Celestia bech32 address. If that's the case, I recommend implementing validation for it.

Reply:
done